src/assets/style.css

```diff
@@ -299,3 +344,28 @@ input[type=radio]:checked + .checkbox {
 .wizard--container-wide {
   width: 80%;
 }
+
+.checkbox-container-disabled {
+  pointer-events: none;
+  background-color: #eaeaea;
+}
+
+.wizard--wip {
+  background-color: rgba(92, 234, 165, 0.08);
+  margin: 10px;
+  padding: 10px;
+}
+
+.wizard--container-wip {
+  padding: 50px;
+}
+
+.wizard--wip-input {
+  background-color: #ffffff;
+  margin: 0 10px 10px 10px;
```

Comment:
Unexpected longhand value '0 10px 10px 10px' instead of '0 10px 10px'           shorthand-property-no-redundant-values